File: users_service/src/main/java/com/mtek/poc/users_service/model/UserModel.java
package com.mtek.poc.users_service.model;


import lombok.*;
import org.springframework.data.annotation.Id;
import org.springframework.data.relational.core.mapping.Column;
import org.springframework.data.relational.core.mapping.Table;

/**
 * @author Emrah TOY
 */
@Data
@ToString
@Builder
@NoArgsConstructor
@AllArgsConstructor
@Table("users.users")
public class UserModel {
    @Id
    @Column("id")
    private Integer id;

    @Column("name")
    private String name;

    @Column("avatar")
    private String avatar;
}

File: user_service/src/main/kotlin/com/mtek/poc/user_service/model/UserModel.kt
package com.mtek.poc.user_service.model

import lombok.*
import org.springframework.data.annotation.Id
import org.springframework.data.relational.core.mapping.Column
import org.springframework.data.relational.core.mapping.Table

@Data
@ToString
@Builder
@NoArgsConstructor
@AllArgsConstructor
@Table("users.users")
open class UserModel(
        @Column
        var username: String?,
        @Column
        var password: String?,
        @Column
        var avatar: String?,
        @Column
        var email: String?,
        @Column
        var first_name: String?,
        @Column
        var last_name: String?,
        @Column
        var uuid: String?,
        @Id
        @Column
        var id: Long?
)